csharp/ql/integration-tests/all-platforms/standalone_dependencies_net48/Assemblies.ql

```diff
@@ -0,0 +1,15 @@
+import csharp
+
+private string getPath(Assembly a) {
+  not a.getCompilation().getOutputAssembly() = a and
+  exists(string s | s = a.getFile().getAbsolutePath() |
+    result =
+      s.substring(s.indexOf("GitHub/packages/") + "GitHub/packages/".length() + 16, s.length())
```

Comment:
Why is `16` also added explicitly (looks like this is also the length of "GitHub/packages/"?

Reply:
We're using a 16-character-long hash as a subfolder name. Let me put these in constants...

Reply:
https://github.com/github/codeql/blob/main/csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs#L283-L284